### ISSUE TITLE
func_static let navmesh go through

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2452]
Test map: compact, coordinates:  780 -1200 -170

Either the navmesh should not be generated there, or blockers should be added. I could not verify if this is a regression since 0.52 or not.

Imo the solution here is to generate navmesh, and put blockers where the entity is. The reasoning behind this is it may be possible for logic entities to enable or not func_static. I do not know much on that topic though, @cu-kai or @illwieckz  will probably know more.

**Comment by cu-kai** [comment]
func_static may not be toggled by other entities (this is the purpose of func_spawn).

**Comment by bmorel** [comment]
I think it's fixed in current master, but no easy way to check there. @cu-kai maybe can be added to a test map when you have time? I know you're busy those day though.

**Comment by necessarily-equal** [comment]
I don't think this is fixed in master yet; but I see you opened very similar PR recently, so maybe that could soon be fixed :)

**Comment by bmorel** [comment]
Clearly not fixed.

**Comment by cu-kai** [comment]
Yes, a single func_static entity can be made up of multiple brushes. Same goes for other func_* entities.

**Comment by sweet235** [comment]
Can we simply handle func_static surfaces the same way as the worldspawn surfaces?

**Comment by slipher** [comment]
Why does the documentation say it is "non-solid"?

https://github.com/Unvanquished/unvanquished-mapeditor-support/blob/99fade39dbc64edb2fbe040a88c61beacfcc2100/src/entities/entities.yaml#L719

**Comment by sweet235** [comment]
Someone found a workaround. Using the `entityLock` command does not change how a func_static entity behaves, but adds a bot obstacle.

For map compact, this prevents the bots from try to go through the func_static walls:

```
changemap compact
setviewpos 1007 -1151 -97 -158 -3
navedit enable human_naked
entityLock 73
entityLock 74
```

One can see that the obstacle's surfaces seem to be restricted to planes parallel to the axis, but that is another story.

Another example:

```
changemap fatal-error
setviewpos 4330 -475 48 3 -23
navedit enable human_naked
entityLock 82
```

**Comment by sweet235** [comment]
> Why does the documentation say it is "non-solid"?

At least that would make a more sense than what we have right now. I see no reason for our current `func_static` to exist, as a map should simply use the worldspawn for these things.

**Comment by sweet235** [comment]
> I see no reason for our current func_static to exist, as a map should simply use the worldspawn for these things.

There are several good reasons. `func_static` is treated differently by map compilers when calculating the lighting. Ocasionally, it can be useful to use `fun_static` to distrupt the bot navmesh, for example to make a passage bots will only traverse in one direction.
